cylc_singleuser.py

```diff
@@ -125,6 +153,10 @@ def start(self):
         signal.signal(signal.SIGINT, app.signal_handler)
         app.listen(self._port)
         ioloop.PeriodicCallback(app.try_exit, 100).start()
+        ioloop.IOLoop.current().add_callback(self.ws_mgr.gather_workflows)
+        ioloop.PeriodicCallback(self.ws_mgr.gather_workflows, 20000).start()
+        ioloop.PeriodicCallback(
+            self.data_mgr.entire_workflow_update, 5000).start()
```

Comment:
May be worth commenting on the choice of these magic numbers?

Reply:
Yeah sure... No particular strategy here, chose 20sec and 5sec respectively, just for testing.. The will change or be removed once PUB/SUB goes in..